src/IdBrokerClient.php

```diff
@@ -434,6 +434,41 @@ public function mfaVerify(string $id, string $employeeId, $value, string $rpOrig
         $this->reportUnexpectedResponse($result, 1506710704);
     }
 
+    /**
+     * Verify an MFA registration (only for webauthn)
+     * @param string $id The MFA ID.
+     * @param string $employeeId The Employee ID of the user with that MFA.
+     * @param string|array $value The MFA value being verified.
+     * @param string $rpOrigin The Relying Party Origin, for WebAuthn MFA options.
+     * @return bool|array
+     * @throws MfaRateLimitException
+     * @throws ServiceException
+     */
+    public function mfaVerifyRegistration(string $id, string $employeeId, $value, string $rpOrigin = '')
+    {
+        $result = $this->mfaVerifyInternal([
+            'id' => $id,
+            'type' => 'registration',
+            'employee_id' => $employeeId,
+            'value' => $value,
+            'rpOrigin' => $rpOrigin,
+        ]);
+        $statusCode = (int)$result[ 'statusCode' ];
+
+        /*
+         * Accept a 204 for compatibility with earlier ID Broker versions
```

Comment:
Do you need to accept a 204 even though earlier versions didn't have this function?